internal/controller/nodedisruption_controller.go

```diff
@@ -34,11 +34,17 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type NodeDisruptionReconcilerConfig struct {
+	// Whether to grant or reject a node disruption matching no node
+	RejectEmptyNodeDisruption bool
```

Comment:
The flag doesn't seem actually used

Reply:
Move the logic within reconcile(), around L100. This part of the function will be moved in its own function in my change to add retry logic. Config would be available directly in the method.

Reply:
Indeed I tested with 2 different builds... If check is moved there we lose the Status object with reason for rejection, OTOH it fits better in ValidateDisruption but config is not available :/